### ISSUE TITLE
avoid prompt() for inputs

### ISSUE THREAD

**Issue by fjenett** [nodebox/live#6]
Avoid use of JS prompt() in general and especially for multi-line inputs as with the stickies.

**Comment by fjenett** [comment]
… that is because it is not well designed in most browsers: the browser UI is much more prominent than the actual label / text to explain what the field asks for. Also it is not consistent across browsers.

**Comment by fdb** [comment]
You're absolutely correct. One issue we discovered is that in recent browsers, you can click a checkbox that will auto-dismiss all prompts, which breaks the UI.

![screen shot 2016-11-29 at 12 17 10](https://cloud.githubusercontent.com/assets/8477/20707910/c7afc574-b62d-11e6-99ab-b56057efc9b0.png)

We're in the process of replacing this with custom modals, and a special UI for the stickies.
